tests/test_track_ensemble.py

```diff
@@ -128,41 +130,52 @@ def test_spatial_perturbations():
     output_directory = DATA_DIRECTORY / 'output' / 'test_original_file'
-    reference_directory = DATA_DIRECTORY / 'reference' / 'test_original_file'
     run_1_directory = output_directory / 'run_1'
     run_2_directory = output_directory / 'run_2'
 
     if not output_directory.exists():
         output_directory.mkdir(parents=True, exist_ok=True)
 
-    reference_track = VortexTrack.from_fort22(reference_directory / 'original.22')
+    original_track_0 = VortexTrack('al062018', start_date='20180911', file_deck='b')
+    original_track_0.to_file(output_directory / 'original.22')
 
     gauss_variables = [MaximumSustainedWindSpeed, CrossTrack]
     range_variables = [RadiusOfMaximumWinds]
 
-    perturber = VortexPerturber(storm='al062018', start_date='20180911', end_date=None)
-
-    perturber.write(
-        perturbations=[-1.0, 1.0], variables=gauss_variables, directory=run_1_directory
+    perturber = VortexPerturber.from_file(
+        output_directory / 'original.22', start_date='20180911'
     )
-    track_1 = VortexTrack.from_fort22(run_1_directory / 'original.22')
 
     perturber.write(
-        perturbations=[-1.0, 1.0], variables=gauss_variables, directory=run_1_directory
+        perturbations=[1.0],
+        variables=gauss_variables,
+        directory=run_1_directory,
+        overwrite=True,
     )
-    track_2 = VortexTrack.from_fort22(run_1_directory / 'original.22')
+    original_track_1 = VortexTrack.from_file(run_1_directory / 'original.22')
 
     perturber.write(
-        perturbations=[-1.0, 1.0], variables=gauss_variables, directory=run_2_directory
+        perturbations=[1.0],
+        variables=gauss_variables,
+        directory=run_2_directory,
+        overwrite=True,
     )
-    track_3 = VortexTrack.from_fort22(run_2_directory / 'original.22')
+    original_track_2 = VortexTrack.from_file(run_2_directory / 'original.22')
 
     perturber.write(
-        perturbations=[-1.0, 1.0], variables=range_variables, directory=run_2_directory
+        perturbations=[1.0],
+        variables=range_variables,
+        directory=run_2_directory,
+        overwrite=True,
     )
-    track_4 = VortexTrack.from_fort22(run_2_directory / 'original.22')
-
-    assert track_1 == reference_track
-    assert track_2 == reference_track
-    assert track_3 == reference_track
-    assert track_4 == reference_track
+    original_track_3 = VortexTrack.from_file(run_2_directory / 'original.22')
+
+    comparison_fields = [
+        field
+        for field in ATCF_FIELDS.values()
+        if field not in ['direction', 'speed', 'extra_values']
+    ]
+    original_data = original_track_0.data[comparison_fields].reset_index(drop=True)
+    pandas.testing.assert_frame_equal(original_track_1.data[comparison_fields], original_data)
+    pandas.testing.assert_frame_equal(original_track_2.data[comparison_fields], original_data)
+    pandas.testing.assert_frame_equal(original_track_3.data[comparison_fields], original_data)
```

Comment:
this test is now updated to use the output `original.22` instead of relying on a reference file. This should test for regression of this bug more thoroughly